packages/frontend/editor-ui/src/plugins/i18n/locales/en.json

```diff
@@ -2993,142 +2993,181 @@
+	"evaluation.list.testRuns": "No test runs | {count} test run | {count} test runs",
+	"evaluation.list.lastRun": "Ran",
+	"evaluation.list.running": "Running",
+	"evaluation.list.errorRate": "Error rate: {errorRate}",
+	"evaluation.list.testStartError": "Failed to start test run",
+	"evaluation.list.testStarted": "Test run started",
+	"evaluation.list.testCancelled": "Test run cancelled",
+	"evaluation.list.loadError": "Failed to load tests",
+	"evaluation.list.item.tests": "No test cases | {count} test case | {count} test cases",
+	"evaluation.list.item.missingFields": "No fields missing | {count} field missing| {count} fields missing",
+	"evaluation.listRuns.status.new": "New",
+	"evaluation.listRuns.status.running": "Running",
+	"evaluation.listRuns.status.evaluating": "Evaluating",
+	"evaluation.listRuns.status.completed": "Completed",
+	"evaluation.listRuns.status.cancelled": "Cancelled",
+	"evaluation.listRuns.status.error": "Error",
+	"evaluation.listRuns.status.success": "Success",
+	"evaluation.listRuns.status.warning": "Warning",
+	"evaluation.listRuns.metricsOverTime": "Metrics over time",
+	"evaluation.listRuns.status": "Status",
+	"evaluation.listRuns.runListHeader": "All runs",
+	"evaluation.listRuns.testCasesListHeader": "Run #{index}",
+	"evaluation.listRuns.runNumber": "Run",
+	"evaluation.listRuns.runDate": "Run date",
+	"evaluation.listRuns.runStatus": "Run status",
+	"evaluation.listRuns.noRuns": "No test runs",
+	"evaluation.listRuns.noRuns.description": "Run a test to see the results here",
+	"evaluation.listRuns.deleteRuns": "No runs to delete | Delete {count} run | Delete {count} runs",
+	"evaluation.listRuns.noRuns.button": "Run Test",
+	"evaluation.listRuns.error.testCasesNotFound": "No matching rows in dataset{description}",
+	"evaluation.listRuns.error.testCasesNotFound.description": "Check any filters or limits set in the evaluation trigger",
+	"evaluation.listRuns.error.executionInterrupted": "Test run was interrupted",
+	"evaluation.listRuns.error.unknownError": "Execution error{description}",
+	"evaluation.listRuns.error.unknownError.description": "Click for more details",
+	"evaluation.listRuns.error.evaluationTriggerNotFound": "Evaluation trigger missing",
+	"evaluation.listRuns.error.evaluationTriggerNotConfigured": "Evaluation trigger is not configured",
+	"evaluation.listRuns.error.evaluationTriggerDisabled": "Evaluation trigger is disabled",
+	"evaluation.listRuns.error.setOutputsNodeNotFound": "No 'Set outputs' node in workflow",
+	"evaluation.listRuns.error.setOutputsNodeNotConfigured": "'Set outputs' node is not configured",
+	"evaluation.listRuns.error.setMetricsNodeNotFound": "No 'Set metrics' node in workflow",
+	"evaluation.listRuns.error.setMetricsNodeNotConfigured": "'Set metrics' node is not configured",
+	"evaluation.listRuns.error.cantFetchTestCases": "Couldn’t fetch test cases{description}",
```

Comment:
is space or `:` needed here? where's the `description` passed? I cannot find where this is passed (same for other errors here).

Reply:
It looks like some commits have been lost through the various rebases... Will try to dig further.
These were intended to display errors together with error details in `TestRunDetailView.vue`.
Will try to dig these out.

Reply:
These are used in for example `TestRunsTable.vue`, look for `<i18n-t>` containing a `<template #description/>` .
In some cases we display the description them with a ". " prepended (when it's part of a phrase), and in some others as a separate element.
This is hacky and ugly, but not sure how to do it differently in vue.

Reply:
I see. thanks for explaining.